libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -407,6 +441,14 @@ export const appStrings = {
     </UiString>
   ),
 
+  instructionsBmdReviewPageChangingVotesPatDevice: () => (
+    <UiString uiStringKey="instructionsBmdReviewPageChangingVotesPatDevice">
+      To change your vote in any contest, use the select button to navigate to
+      that contest. When you are finished making your ballot selections and
+      ready to print your ballot, use the right button to print your ballot.
```

Comment:
I think "right button" is left over from the ATI controller instructions.

Reply:
Thanks for catching - forgot to modify this string